autocomplete-example-yaml/README.md

```diff
@@ -0,0 +1,40 @@
+# Demo
+
+It shows how `monaco-editor` and `monaco-yaml` can be used with
+our `blueprint-schema.json` to edit/validate the syntax.
+
+## Table of Contents
+
+- [Prerequisites](#prerequisites)
+- [Setup](#setup)
+- [Running](#running)
+
+## Prerequisites
+
+- [NodeJS](https://nodejs.org) 16 or higher
+- [npm](https://github.com/npm/cli) 8.1.2 or higher
+
+## Setup
+
+To run the project locally, clone the repository and set it up:
+
+```sh
+git clone https://github.com/lzap/common-blueprint-example
+cd common-blueprint-example
+```
+
+## Running
+
+To start it, simply run:
+
+```sh
+make run-web-editor-yaml
+```
+
+The demo will open in your browser.
+
+# Kudos
+This whole demo setup was shamelessly copied
+from https://github.com/remcohaszing/monaco-yaml/tree/main/examples/demo
```

Comment:
Can you include the LICENCE from the source and change this "shameless" statement to just a regular "this was copied from XYZ, thank you"? Forking/copying is the lifeblood of FOSS, you are doing nothing wrong just keep the copyright statement in the license and all is good.